.github/workflows/test_branches.yml

```diff
@@ -107,16 +99,12 @@ jobs:
             echo "GHA_JOBGROUP=other" >> $GITHUB_ENV
         fi
         # Note: pandas 1.0.3 causes gams 29.1.0 import to fail in python 3.8
-        PYTHON_PACKAGES="${PYTHON_REQUIRED_PKGS}"
+        EXTRAS=tests
```

Comment:
Alternative way to do this: in the `env` section, set `EXTRAS=test,docs,optional` and then for `slim`, change that. I have no strong opinions, but it's fewer lines.